Alloy/Encoders/MinEncoder.swift

```diff
@@ -39,7 +39,7 @@ public class MinEncoder {
             encoder.setThreadgroupMemoryLength(threadgroupSize.width * threadgroupSize.height * 4 * MemoryLayout<Float16>.stride,
                                                index: 0)
             encoder.dispatch2d(state: self.pipelineState,
-                               covering: MTLSize(width: 1, height: 1, depth: 1),
+                               covering: MTLSize(repeating: 1),
```

Comment:
I like it, but I also think we can introduce static thing like `.one` or `.identity` for convenience, what you think?

Reply:
We can add these extra extensions that use repeating: inside

Reply:
yup, of course